src/Loop.php

```diff
@@ -59,155 +59,162 @@ public static function stop()
     /**
      * Defer the execution of a callback.
      *
-     * @param callable $callback The callback to defer.
+     * @param callable(mixed $data, string $watcherIdentifier) $callback The callback to defer.
```

Comment:
Even with object watchers, it's somewhat weird, IMO.

It doesn't exactly hurt. Just as said, $data is already covering it and more logical to me.